Themes/default/languages/EmailTemplates.english.php

```diff
@@ -319,8 +319,7 @@
 $txt['alert_unapproved_post_subject'] = 'New Unapproved Post: {SUBJECT}';
 $txt['alert_unapproved_post_body'] = 'A new post has been made which needs to be approved: \'{SUBJECT}\'
 
-You can approve or reject this post by clicking the link below:
-
+You can approve or reject this post on the link below:
```

Comment:
i dont agree with this change. you need the part about clicking on it to make sense

Reply:
The point here is to change the "by" because it could be interpreted as "you will do an action by clicking the link", when the purpose is to say you can do said action "after following the link". Any other suggestions to have that effect?